packages/surface/src/index.js

```diff
@@ -0,0 +1,10 @@
+import "@hig/styles/build/fonts.css";
```

Comment:
Should this be here? I didn't see in the `package.json` that we are outputting a css file so this won't get included for the user (also no mention of it in the readme). The `fonts` package isn't listed as a dependency either.